tango/workspaces/local_workspace.py

```diff
@@ -72,7 +74,7 @@ def __init__(self, dir: PathOrStr):
             with SqliteDict(self.step_info_file) as d:
                 for stepinfo_file in self.cache.dir.glob("*/stepinfo.dill"):
                     with stepinfo_file.open("rb") as f:
-                        stepinfo = StepInfo.deserialize(f.read())
+                        stepinfo = dill.load(f)
```

Comment:
It's being used by the `WandbWorkspace`. I think it would be better to store step info here as JSON as well, but that would be a breaking change and I wanted to limit the scope of this PR as best I could.